types/react-native/index.d.ts

```diff
@@ -8500,6 +7860,12 @@ export interface UIManagerStatic {
         success: (item: string, index: number | undefined) => void
     ): void;
 
+    getViewManagerConfig: (
+        name: string,
+      ) => {
+        Commands: {[key: string]: number};
```

Comment:
I think this might be optional, but it is typed as `Object` in the RN flow type, so...this is at least better than that...

I'm fine with this if you are.